### PR TITLE
form.action returns the location URL for a form with no action attribute set (fix #268)

### LINKED ISSUE #268: form.action returns the location URL for a form with no action attribute set
Form without attributes:
```html
<form></form>
```
JS:
```javascript
console.log(form.action); // in ie edge - location url, in other browsers - empty string

console.log(__get$(form, 'action')); // in all browsers - empty string

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @inikulin @churkin

### CODE REVIEW

**Comment by inikulin** [comment]
I don't see feature detection here and I don't see regression test for case described in the issue.

\r-

**Comment by LavrovArtem** [comment]
@inikulin How does the feature detection look like?

**Comment by LavrovArtem** [comment]
@inikulin Test "test/client/fixtures/sandbox/node/attributes-test.js" fail in ie edge, this commit fix it.
image

**Comment by LavrovArtem** [comment]
@inikulin This error does not reproduce on sauce lab.

**Comment by LavrovArtem** [comment]
ping @inikulin @churkin

**Comment by inikulin** [comment]
@LavrovArtem I don't understand the fix and I don't get how it solves Edge-specific problem. Let's discuss in person tomorrow.

**Comment by LavrovArtem** [comment]
@inikulin ok

**Comment by AlexanderMoskovkin** [comment]
@LavrovArtem you can use the build-bot command to restart the tests instead of reopening the issue:
`@testcafe-build-bot \retest`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -45,16 +45,19 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
     static _getUrlAttr (el, attr) {
         var attrValue = el.getAttribute(attr);
 
-        if (attrValue === null)
-            return '';
+        // NOTE: In some browsers, elements without the url attribute return the location url
+        // when accessing this attribute directly. See form.action in Edge 25 as an example.
+        var browserFallsBackToLocationUrl = attrValue === null && el[attr];
```
Comment (inikulin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: Can we make feature detection here, e.g.:
*src/client/feature-detections.js*
```js
 export.emptyUrlAttrFallbacksToTheLocation = NativeMethods.createElement(document, 'a').href === window.location;
```

here:
```
if (attrValue === '' && featureDetection.emptyUrlAttrFallbacksToTheLocation)
    return destLocation.get();		             
```

**Comment by inikulin** [comment]
\r-

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by inikulin** [comment]
nice, lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**